clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java

```diff
@@ -358,7 +358,7 @@ public void sendHeartBeats(HeartbeatsRequest heartbeatsRequest) {
   public boolean sendHeartBeat(String serviceId, String instanceId) {
     try {
       HttpResponse response = httpClient
-          .putHttpRequest("/registry/microservices/" + serviceId + "/instances/" + instanceId + "/heartbeat",
+            .putHttpRequest("/registry/microservices/" + serviceId + "/instances/" + instanceId + "/heartbeat",

```

Comment:
wrong indention. should not change this.

Reply:
done